.changes/b1c5ee5a-650f-4499-95ee-c891e2703657.json

```diff
@@ -0,0 +1,8 @@
+{
+    "id": "b1c5ee5a-650f-4499-95ee-c891e2703657",
+    "type": "feature",
+    "description": "Add binary compatibility validator",
+    "issues": [
+        "https://github.com/awslabs/smithy-kotlin/issues/598"
+    ]
+}
```

Comment:
**Comment**: I question whether this deserves a changelog entry. It's not a user-facing feature—the primary beneficiaries are developers. Furthermore, it won't be "included" in the next release—it will be active as soon as it's merged.